datalad/support/tests/test_annexrepo.py

```diff
@@ -928,6 +917,39 @@ def test_annex_ssh(repo_path, remote_1_path, remote_2_path):
     ok_(exists(socket_2))
 
 
+@with_testrepos('basic_annex', flavors=['clone'])
+@with_tempfile(mkdir=True)
+def test_annex_remove(path1, path2):
+    ar1 = AnnexRepo(path1, create=False)
```

Comment:
Don't get it by now. Locally, it's working for me. Meaning: The test is actually running.